techniques/failures/F92.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html><html lang="en"><head><title>Failure of Success Criterion 1.3.1 due to the use of role presentation on content which conveys semantic information</title><link rel="stylesheet" type="text/css" href="../../css/editors.css" class="remove"></head><body><h1>Failure of Success Criterion 1.3.1 due to the use of role presentation on content which conveys semantic information</h1><section class="meta"><p class="id">ID: F92</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
       <p>HTML</p>
    </section><section id="description"><h2>Description</h2>
-      <p>This failure occurs when a role of presentation is applied to an element whose purpose is to convey information or relationships in the content. Elements such as <code class="language-html">table</code>, can convey information about the content contained in them via their semantic markup. The WAI-ARIA role of <a href="https://www.w3.org/TR/wai-aria/#presentation">
-            <code class="language-html">presentation</code>
-         </a> on the other hand, is intended to suppress semantic information of content from the accessibility API and prevent user agents from conveying that information to the user. Use of the <code class="language-html">presentation</code> role for content which should convey semantic information may prevent the user from understanding that content.
+      <p>This failure occurs when a role of presentation is applied to an element whose purpose is to convey information or relationships in the content. Elements such as <code>table</code>, can convey information about the content contained in them via their semantic markup. The WAI-ARIA role of <a href="https://www.w3.org/TR/wai-aria/#presentation">
+            <code>presentation</code>
+         </a> on the other hand, is intended to suppress semantic information of content from the accessibility API and prevent user agents from conveying that information to the user. Use of the <code>presentation</code> role for content which should convey semantic information may prevent the user from understanding that content.
```

Comment:
```suggestion
         </a> on the other hand, is intended to suppress semantic information of content from the accessibility API and prevent user agents from conveying that information to the user. Use of <code>role="presentation"</code> for content which should convey semantic information may prevent the user from understanding that content.
```

could then benefit from highlighting